learningresources/tests/test_views.py

```diff
@@ -0,0 +1,54 @@
+"""
+Test the importer views to make sure they work.
+"""
+
+from __future__ import unicode_literals
+
+from django.test import Client
+from django.test.testcases import TestCase
+from django.contrib.auth.models import User
+
+from learningresources.models import Repository
+
+HTTP_OK = 200
+
+
+class TestViews(TestCase):
+    """Hit each view."""
+
+    def setUp(self):
```

Comment:
This is going to be such a common pattern I think we should run this out to a base class either in each app if we want to keep these apps de-composable, or into the project, and inherit from anywhere that needs a user in their tests.  I also think this would be cleaner as a one-liner ala:
```python
self.user = User.objects.create_user(
            username='test', password='test_pass'
)
```